cli/commands/remote/create/table/index.md

```diff
@@ -6,29 +6,27 @@ permalink: /cli/commands/remote/create/table/
 * TOC
 {:toc}
 
-**Create a [table](https://help.keboola.com/storage/tables/) in Keboola Storage.**
+## Create a Table from the CLI
+
+To create a [table](https://help.keboola.com/storage/tables/) in Keboola Storage directly from the command line interface, use the following command:
 
 ```
 kbc remote create table [flags]
 ```
 
 ### Options
 
-`--bucket <string>`
-: Bucket ID
-
-`--columns <string>`
-: Comma-separated list of column names
-
-`--name <string>`
-: Name of the table
-
-`--primary-key <string>`
-: Comma-separated list of columns used as a primary key
+- `--bucket <string>`: Specifies the bucket ID where the table will be created.
+- `--columns <string>`: Defines a comma-separated list of column names for the table.
+- `--columns-from <string>`: Indicates the path to the table definition file in JSON format.
+- `--name <string>`: Sets the name of the new table.
+- `--primary-key <string>`: Determines a comma-separated list of columns to be used as the primary key.
```

Comment:
@hhanova hi, this your change breaks the visual aspect of the page and readability

-------------------

Before: 
![image](https://github.com/keboola/developers-docs/assets/19371734/fade436a-59ca-4bd5-a36b-bee64568658e)


--------------

After:

![image](https://github.com/keboola/developers-docs/assets/19371734/fe816d3b-b290-40e9-a410-4532e62fb2f3)

Reply:
I don't agree that these changes break readability... It's just different :-).

For now, I'll leave it the way the other command sections were done (didn't know about that; see the comment below).

Reply:
> I was not asked to proofread the other (older) commands in the developer docs, so I had no idea they were done differently. Sorry about that. I should have looked.

You were :slightly_smiling_face:  you did a review on it, but we haven't changed anything there for a long time.

Ok
- It is now not consistent with the other CLI commands (the main issue).
- Elsewhere in the documentation, we don't have CLI flags, so it's probably hard to compare it with something else.
- You are primarily interested in the names of the flags, description is secondary, that reflects the original design.
- The CLI will probably have a separate documentation page, automatically generated.
- So we will revert the list of flags back. Thanks!

Reply:
I reversed the changes you did not like. The descriptions were still secondary, but I understand it should be consistent with the other pages. 

As for me doing a review of other commands... You might be right. I could have just forgotten about that :-). But that does not change the fact that there are command pages that have not been proofread. I'll do it asap.